FlowCrypt/Controllers/Setup/SetupViewDecorator.swift

```diff
@@ -80,7 +80,7 @@ struct SetupViewDecorator {
         case .common:
             subtitle = "setup_description".localized
         case .choosingPassPhrase:
-            subtitle = "" // "create_pass_phrase_description".localized https://github.com/FlowCrypt/flowcrypt-ios/issues/497
+            subtitle = "create_pass_phrase_description".localized
```

Comment:
`See choosing secure pass phrase ?` looks strange, at first I thought that it's incorrect localisation string, and it's not clear that you can tap on it.

Let's change text to `Tap to see tips for choosing secure pass phrase`